http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -164,6 +166,35 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
       )
     } yield ()
 
+  private def generateAzureVMSecurePassword(): String = {
```

Comment:
I kinda wanted to add a unit test for this, but not quite sure if this is the best location for this function in the first place. There does not seem to be a kind of `utils` module for leo that i could stick it into

Reply:
I heard an interesting philosophy recently that `utils` are generally a smell and are an easy way to end up with dependency spaghetti (*point to UI dependency graph which has tons of utils*). Rather, it is better for util methods to live in the file they are consumed, especially if there is only one such file.

Reply:
Also I just realized this is totally in a `util` folder 🤣

Reply:
(in the interest of adding to this conversation) - I agree with @jdcanas re: smell (the single file ends up being a dumping ground), and also, I've also read that an alternative is to having a single file is to break out the file into parts and have one "util" aptly named file that reflects the method it represents.